src/rules/converters/jsdoc-format.ts

```diff
@@ -0,0 +1,22 @@
+import { RuleConverter, ConvertedRuleChanges } from "../converter";
+
+export const JSDocNoticeMsg =
+    "ESLint does not support enforcing the first line of multiline JSDoc comments be empty.";
+
+export const convertJSDocFormat: RuleConverter = () => {
+    const ruleNames = [
+        "jsdoc/check-alignment",
+        "jsdoc/check-indentation",
+        "jsdoc/newline-after-description",
+    ];
+
+    const mappedRuleNames: ConvertedRuleChanges[] = ruleNames.map(ruleName => {
+        return { ruleName };
+    });
+
+    return {
+        rules: [...mappedRuleNames],
+        notices: [JSDocNoticeMsg],
+        plugins: ["eslint-plugin-jsdoc"],
```

Comment:
@JoshuaKGoldberg I think it should be `jsdoc` instead of `eslint-plugin-jsdoc` as stated on [eslint-plug-jsdoc configuration](https://github.com/gajus/eslint-plugin-jsdoc/blob/master/README.md#configuration) docs.

Reply:
Hmm, the ESLint docs request the `eslint-plugin-` prefix: https://eslint.org/docs/developer-guide/working-with-plugins

I have a vague memory of either working? So either way is probably fine 🤷‍♂